src/app/static/src/app/store/adr/actions.ts

```diff
@@ -154,9 +155,21 @@ export const actions: ActionTree<ADRState, RootState> & ADRActions = {
             const { resourceType, resourceFilename, resourceId } = uploadFilesPayload[i]
 
             const requestParams: Dict<string> = {resourceFileName: resourceFilename}
-            if (resourceId){
+            if (resourceId) {
                 requestParams["resourceId"] = resourceId
             }
+            if (uploadMetadata && resourceType === state.schemas?.outputSummary) {
+                requestParams["description"] = uploadMetadata.outputSummary.description
+            } else if (uploadMetadata && resourceType === state.schemas?.outputZip) {
+                requestParams["description"] = uploadMetadata.outputZip.description
+            } else {
+                /**
+                 * Default description will tentatively solve backward
+                 * compatibility issue that may arise with previously calibrated model
+                 * and once re-calibrated, the correct data will be populated
+                 */
+                requestParams["description"] = "Output upload from Naomi web"
```

Comment:
Should this match [the Naomi boilerplate](https://github.com/mrc-ide/naomi/blob/master/R/run-model.R#L657-L664)? Note that it distinguishes between "output" and "summary report": we should probably do that, at least. For example:
```kotlin
if (resourceType === state.schemas?.outputSummary) {
   requestParams["description"] = uploadMetadata
        ? uploadMetadata.outputSummary.description
        : "Naomi summary report uploaded from Naomi web app"
}
```

Reply:
I like the idea so I tried it at first but not working well. I will create a ticket to adopt it.

Reply:
Do you remember what the problem was? Or have a link to ticket?

Even if this is just a fallback it would be better not to attach an identical description to both output files and summary reports. Ideally "Naomi output uploaded from Naomi web app" and "Naomi summary report uploaded from Naomi web app" respectively, for consistency with descriptions attached by Naomi.

Reply:
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-2367

Reply:
This is now updated in this ticket.